include/deal.II/differentiation/sd/symengine_scalar_operations.h

```diff
@@ -36,6 +39,8 @@ namespace Differentiation
 {
   namespace SD
   {
+    namespace SE = ::SymEngine;
```

Comment:
I come late to this party, but doing this in a header file and using the abbreviation makes it unnecessarily hard for readers of the documentation to understand where a name comes from because it lists the wrong namespace name in the documentation (it may *link* to the correct name, but it doesn't show it). If you wanted to spend 30 seconds to use the original namespace name everywhere below, then I'll pre-approve such a patch :-)

Reply:
Makes sense. I'll do this throughout the headers I've added thus far.